app/src/main/java/com/google/homesampleapp/data/UserPreferencesRepository.kt

```diff
@@ -62,6 +62,18 @@ class UserPreferencesRepository @Inject constructor(@ApplicationContext context:
     }
   }
 
+  suspend fun isShowHalfsheetNotification(): Boolean {
```

Comment:
nit: rename to shouldShowHalfsheetNotification

Reply:
Done.